types/FBLoginButton.d.ts

```diff
@@ -57,6 +57,11 @@ export class LoginButton extends React.Component<{
      * View style, if any.
      */
   style?: any;
+
+  /**
+   * testID, if any.
+   */
+  testID?: string,
```

Comment:
Oh of course, I should have seen that yesterday when I reviewed your implementation PR. Thanks for the follow-up

I think it needs to be like this though? It looks like we are inconsistent on this point actually! Most are semicolon-terminated (correct I think) while some are commas. I wonder if our types are being tested correctly :thinking: . Do you confirm this works for you? Can you confirm it works with the semi-colon (which I think is more correct)?

```suggestion
  testID?: string;
```

Reply:
Good catch, thank you. 

I had tested it locally with the correct fix but somehow submitted this... hmmm.